### PR TITLE
Roles select on development

### PR DESCRIPTION
**Pull request by carmeltsadok**

### CODE REVIEW

File: src/pages/QuestionnaireSchemaPage/components/Header/Header.styles.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const useStyles = makeStyles((theme) => ({
     width: '100%',
     height: '10%',
     minHeight: '5rem',
+    maxHeight: '5rem',
```
Comment (lustigy96) [inline src/pages/QuestionnaireSchemaPage/components/Header/Header.styles.js:9]:
```suggestion
height: '5rem'
```

File: src/pages/NodeGroupPage/components/Header/Header.styles.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ const useStyles = makeStyles((theme) => ({
     width: '100%',
     height: '10%',
     minHeight: '5rem',
+    maxHeight: '5rem',
```
Comment (lustigy96) [inline src/pages/NodeGroupPage/components/Header/Header.styles.js:9]:
```suggestion
    height: '5rem',
```

File: src/common/Header/Header.styles.js [hunk 3]
```diff
@@ -35,6 +35,13 @@ const useStyles = makeStyles((theme) => ({
   secondary: {
     color: theme.palette.primary.secondary,
   },
+  select: {
+    backgroundColor: 'white',
+    width: '15rem',
+    margin: '0 0.5rem',
+    borderRadius: '15px',
+    padding: '0.2rem 0',
```
Comment (lustigy96) [inline src/common/Header/Header.styles.js:43]: on reports page - add margin-bottom in sub-header (just so it, although it is not here in the changed files) 
also, this header select is ugly (haha) \\ just for the next time 
make it like the select in the reports page. maybe add a common css configuration

File: src/common/Header/Header.jsx [hunk 4]
```diff
@@ -24,6 +36,13 @@ const Header = () => {
     }
     return t('headerBlessings.night');
   };
+
+  useEffect(() => {
+    if (selectedRole.id !== UserStore.userProfile.role) {
+      AuthService.reconnect(selectedRole.id);
+    }
+  }, [selectedRole]);
+
```
Comment (lustigy96) [inline src/common/Header/Header.jsx:45]: only in development state

**Review by lustigy96 — CHANGES_REQUESTED** [review]

**Review by maykremer — APPROVED** [review]

**Review by PCOffline — CHANGES_REQUESTED** [review]
Instead of using `const` and then immediately returning it, put the expression inside the

**Review by lustigy96 — APPROVED** [review]

**Pull request merged**